### PR TITLE
bugfix: Fix hails for disabled hostile ships when the player requires assitance

### PR DESCRIPTION
**Pull request by tibetiroka**
**Bugfix:** This PR addresses an issue reported by @Terin on discord: hostile ships show friendly hails to the player if they are disabled, and the player needs help.

## Fix Details
Hostile ships now handle both disabled and non-disabled cases.

## Testing Done
I flew around and checked the new hail.

### CODE REVIEW

File: source/HailPanel.cpp [hunk 1]
```diff
@@ -62,8 +62,11 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy() && !ship->IsDisabled())
-		SetBribe(gov->GetBribeFraction());
+	else if(gov->IsEnemy())
```
Comment (RisingLeaf) [inline source/HailPanel.cpp:65]: Small comment explaining the new cases please.

**Review by RisingLeaf — APPROVED** [review]
Looks fine, just one small thing

File: source/HailPanel.cpp [hunk 2]
```diff
@@ -62,8 +62,14 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
...
```
Comment (quyykk) [inline source/HailPanel.cpp:65]: Can you add braces to the if? thanks. It looks weird otherwise

File: source/HailPanel.cpp [hunk 3]
```diff
@@ -62,8 +62,16 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
...
+	{
+		// Enemy ships always show hostile messages.
+		// They either show bribing messages,
+		// or standard hostile messages, if disabled.
+		if(ship->IsDisabled())
+			message = ship->GetHail(player.GetSubstitutions());
+		else
+			SetBribe(gov->GetBribeFraction());
```
Comment (warp-core) [inline source/HailPanel.cpp:73]: `message = ship->GetHail(player.GetSubstitutions());` is run after this if block if `message` is empty, so don't run the same line here.
```suggestion
		if(!ship->IsDisabled())
			SetBribe(gov->GetBribeFraction());
```

**Review by warp-core — APPROVED** [review]

**Pull request merged**